### PR TITLE
Not clean up "config" and "config_raw" in error handling code

### PR DESCRIPTION
**Pull request by bigdata-memory**
The config variable points to a new allocated memory block inside of load_manifest() but it isn't cleaned up on two error handling branches.
also the mapped config_raw is not unmapped on last error handling branch before return.

---
This change is [Reviewable](https://reviewable.io/reviews/oscarlab/graphene/248)

### CODE REVIEW

**Comment by donporter** [comment]
Ok to test

File: Pal/src/host/Linux-SGX/sgx_main.c [hunk 1]
```diff
@@ -644,6 +646,8 @@ int load_manifest (int fd, struct config_store ** config_ptr)
 
     if (ret < 0) {
         SGX_DBG(DBG_E, "can't read manifest: %s\n", errstring);
+        INLINE_SYSCALL(munmap, 2, config_raw, nbytes);
+        free(config);
```
Comment (donporter) [inline Pal/src/host/Linux-SGX/sgx_main.c]: I think this function is right on the cusp of needing a single error case at the end with gotos.  I think the code is ok as-is, but I think one more system call/cleanup case would tip this into having one exception case at the end.

**Comment by bigdata-memory** [comment]
Sure, I will do it. thanks.

**Comment by bigdata-memory** [comment]
Hi @rainfld and @donporter, your suggestions have been merged to this PR, Please review it again, thanks.

**Comment by donporter** [comment]
I think this can be merged on Sunday or when it gets a third approval.

**Comment by donporter** [comment]
Test this please

**Pull request merged**